message/message1_1prime/message.go

```diff
@@ -219,16 +181,16 @@ func FromNet(r io.Reader) (datatransfer.Message, error) {
 }
 
 // FromNet can read a network stream to deserialize a GraphSyncMessage
-func FromIPLD(node datamodel.Node) (datatransfer.Message, error) {
+func FromIPLD(node ipld.Node) (datatransfer.Message, error) {
```

Comment:
My understanding is that datamodel.Node was the "updated" place to refer to these as the rest were just type aliases for compatibility with older versions.

Reply:
yeah, I don't think it matters too much but Eric would probably be happier if I didn't use it like this.